plugins/cua/vendor/cua-driver/source/Tests/integration/conftest.py

```
@@ -0,0 +1,282 @@
+    app_path = os.path.join(_ELECTRON_APP_DIR, _ELECTRON_APP_NAME)
+    _download_and_extract(_ELECTRON_DOWNLOAD_URL, _ELECTRON_APP_DIR, app_path)
+
+    if not os.path.exists(app_path):
+        pytest.skip("Electron app not found after download")
+
+    test_page_url = f"{html_server}/test_page.html"
+
+    subprocess.run(["pkill", "-f", "desktop-test-app-electron"], check=False)
+    time.sleep(0.5)
+
+    subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)